ui/raidboss/data/00-misc/test.js

```diff
@@ -242,6 +243,33 @@ export default {
         };
       },
     },
+    {
+      id: 'Test Watch',
+      netRegex: NetRegexes.echo({ line: 'cactbot test watch.*?', capture: false }),
+      promise: (data) => Util.watchCombatant({ names: [data.me, 'Striking Dummy'] }, (ret) => {
```

Comment:
Can you put a `LocaleText` of striking dummy names at the top of the file so this can be translated?